packages/cli-lib/lang/en.lyaml

```diff
@@ -22,6 +22,20 @@ en:
                 describe: "New name for account."
             success:
               renamed: "Account \"{{ name }}\" renamed to \"{{ newName }}\""
+          setDefault:
+            describe: "Change default account"
+            errors:
+              accountNotFound: "The account \"{{ specifiedAccount }}\" was not found in {{ configPath }}."
+            examples:
+              default: "Select account to use as the default from a list"
+              idBased: "Set the default account to the account in the config with accountId equal to \"1234567\""
```

Comment:
Will the accountId `1234567` potentially be confusing to any customers?

Reply:
It's what was there before. Do you have an idea for a better example accountId?

Reply:
Ah, if that was the example before, then I think it's good to go!